### PR TITLE
Update description template and use latest alias

### PR DESCRIPTION
**Pull request by farshidtz**

### CODE REVIEW

File: metadata/edgex-device-usb-camera.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This service provides the following capabilities:
 
 **Getting started**
 
-https://docs.edgexfoundry.org/2.3/getting-started/Ch-GettingStartedSnapUsers/#device-usb-camera
+https://docs.edgexfoundry.org/latest/getting-started/Ch-GettingStartedSnapUsers/#device-usb-camera
```
Comment (MonicaisHer) [inline metadata/edgex-device-usb-camera.md:12]: device-usb-camera is only available since 2.3

Comment (MonicaisHer) [reply metadata/edgex-device-usb-camera.md:12]: ok, 2.3 will become the latest/stable version after this release.

File: metadata/edgex-device-grove.md [hunk 2]
```diff
@@ -7,14 +7,17 @@ This device service is expected to work on the following boards:
 * Raspberry Pi 3 or higher - arm64 OS
 * Raspberry Pi Zero 2 or higher - arm64 OS (untested)
 
-**Source code and more info**
-
-https://github.com/edgexfoundry/device-grove-c
+**Getting started**
 
+https://docs.edgexfoundry.org/latest/getting-started/Ch-GettingStartedSnapUsers/#device-grove
 **EdgeX documentation**
```
Comment (MonicaisHer) [inline metadata/edgex-device-grove.md:13]:
```suggestion
https://docs.edgexfoundry.org/latest/getting-started/Ch-GettingStartedSnapUsers/#device-grove

**EdgeX documentation**
```

File: metadata/edgexfoundry.md [hunk 3]
```diff
@@ -10,34 +10,30 @@ This snap is a part of EdgeX Foundry and contains the following reference servic
...
 
 **Supporting Services**
-* Rules Engine (eKuiper)
+* Rules Engine (eKuiper) - Deprecated in favor of https://snapcraft.io/edgex-ekuiper
 * Scheduling
 * Alerts & Notifications
 
 **Application Services**
-* Configurable Application Service
+* Configurable Application Service (for Rules Engine event filtering) - Deprecated in favor of https://snapcraft.io/edgex-app-service-configurable
```
Comment (MonicaisHer) [inline metadata/edgexfoundry.md:22]: ASC is not only for rules engine event filtering but also for http-export, mqtt-export etc.
https://github.com/edgexfoundry/app-service-configurable/tree/main/res

Comment (farshidtz) [reply metadata/edgexfoundry.md:22]: The one embedded in edgexfoundry app is for rules engine only. The hooks make sure of that.

**Review by MonicaisHer — APPROVED** [review]
@farshidtz Thank you, looks good!

**Pull request merged**